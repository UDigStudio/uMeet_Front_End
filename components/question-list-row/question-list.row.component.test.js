import React from 'react';
import QuestionListRow from './question-list-row.component';
import renderer from 'react-test-renderer';

describe('Question List Row', () => {
  test('Renders a question', () => {
    const component = renderer.create(
      <QuestionListRow 
        question={{ id: '1', text: 'First question?', active: true }} 
      />
    );
    let tree = component.toJSON(); 
    expect(tree).toMatchSnapshot();
  })
})