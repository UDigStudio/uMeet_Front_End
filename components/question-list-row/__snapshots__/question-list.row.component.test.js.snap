// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question List Row Renders a question 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "marginBottom": 10,
      "marginLeft": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "marginRight": 10,
      }
    }
  >
    First question?
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#39FF14",
        "borderRadius": 5,
        "height": 10,
        "width": 10,
      }
    }
  />
</View>
`;
